### ISSUE TITLE
Uncaught TypeError: Cannot read property 'range' of undefined

### ISSUE THREAD

**Issue by dwelle** [paradoxxxzero/next-occurrence#2]
`next-occurrence:next`/`prev` throws an error.

**Atom Version**: 1.5.0-beta0
**System**: Windows 7
**Thrown From**: [next-occurrence](https://github.com/paradoxxxzero/next-occurrence) package, v0.4.0
### Stack Trace

Uncaught TypeError: Cannot read property 'range' of undefined

```
At /.atom/packages/next-occurrence/lib/next-occurrence.coffee:69

TypeError: Cannot read property 'range' of undefined
    at Object.module.exports.prev (file:///.atom/packages/next-occurrence/lib/next-occurrence.coffee:47:26)
    at atom-text-editor.disposables.add.atom.commands.add.next-occurrence:prev (file:///.atom/packages/next-occurrence/lib/next-occurrence.coffee:15:10)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (\atom\app-1.5.0-beta0\resources\app.asar\src\command-registry.js:260:29)
    at \atom\app-1.5.0-beta0\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (\atom\app-1.5.0-beta0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (\atom\app-1.5.0-beta0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (\atom\app-1.5.0-beta0\resources\app.asar\src\window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (\atom\app-1.5.0-beta0\resources\app.asar\src\window-event-handler.js:3:61)
```

**Comment by olmokramer** [comment]
Just published a new version 0.5.0 that should fix this. Please let me know if it works

**Comment by dwelle** [comment]
Works.
